db/migrate/20220915123753_change_user_ref_to_post.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    remove_reference :posts, :user, null: false, foreign_key: true
+    add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
+  end
```

Comment:
- [ ] this migration file also failing, it says 
```
Table 'comments' has no foreign key for users
/Users/tbilisi/Documents/4. Microverse/1. reviews/Blog-App/db/migrate/20220915123925_change_user_ref_to_comments.rb:3:in `change'
``` 
which means that you should be removing author reference not user, and delete foreign_key, this will work:

```
  def change
    remove_reference :posts, :author
    add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
  end
```